packages/server/modules/blobstorage/tests/blobstorage.integration.spec.js

```diff
@@ -144,18 +93,38 @@ describe('Blobs integration @blobstorage', () => {
     email: 'barron@bubble.bobble',
     password: 'bubblesAreMyBlobs'
   }
+  const workspace = {
```

Comment:
with only these changes this entire test suite now runs in both non-multiregion and multiregion mode, and actually properly stores data (both db rows & blobs) into region storage